internal/service/billing.go

```diff
@@ -186,11 +186,22 @@ func (s *billingService) CalculateUsageCharges(
 	priceService := NewPriceService(s.ServiceParams)
 
 	// Process usage charges from line items
+	activeLineItems := make([]*subscription.SubscriptionLineItem, 0)
```

Comment:
lets clean this up as well so that service is not worried about active line items